apps/web/src/design-system/icons/Icons.stories.tsx

```diff
@@ -29,7 +29,7 @@ export const Icons = () => {
           if (name.includes(search.toLowerCase())) {
             return (
               <Grid.Col key={name} span={3}>
-                <Group align="center" spacing={3} direction="row">
+                <Group align="center" spacing={3} style={{ flexDirection: 'row' }}>
```

Comment:
```suggestion
                <Group align="center" spacing={3}>
```
not needed as it has row as default direction